.changeset/little-ads-return.md

```diff
@@ -0,0 +1,33 @@
+---
+"@makeswift/runtime": minor
+---
+
+BREAKING: The latest upgrade to the Makeswift client `getPages` method
+introduces sorting, path filtering, and pagination. This method was not
+previously paginated - in order to get all your pages, you must now paginate
+through results:
+
+```tsx
+import { client } from '@/makeswift/client'
+import { MakeswiftPageResult } from '@makeswift/runtime/next'
+
+async function getAllPages(): Promise<MakeswiftPageResult[]> {
+  let hasMorePages = true
+  let after: string | undefined = undefined
+  const pages: MakeswiftPageResult[] = []
+
+   do {
+    const { data, hasMore } = await client.getPages({ after })
```

Comment:
I think the awkwardness of this example probably calls for the async iterator protocol @migueloller has been advocating for.